templates/python/custom-copilot-rag-customize/env/.env.local.user.tpl

```diff
@@ -0,0 +1,28 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
+
```

Comment:
Do we need to include `.env.local` and `.env.testtool` files in the template? I only see the `.user` file for them.

Reply:
yes we do. I'll add them.